ruby_event_store_rom_sql/ruby_event_store_rom_sql.gemspec

```diff
@@ -31,8 +31,8 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'google-protobuf', '~> 3.5.1.2'
 
   spec.add_dependency 'ruby_event_store', '= 0.27.1'
-  spec.add_dependency 'sequel', '~> 4.49'
-  spec.add_dependency 'rom-sql', '~> 2.4.0'
-  spec.add_dependency 'rom-repository', '~> 2.0.2'
-  spec.add_dependency 'rom-changeset', '~> 1.0.2'
+  spec.add_dependency 'sequel', '>= 4.49'
```

Comment:
any particular reason why? assuming ROM is sem-ver why not 

```
spec.add_dependency 'rom-sql', '~> 2.4'
spec.add_dependency 'rom-repository', '~> 2.0'
spec.add_dependency 'rom-changeset', '~> 1.0'
```

etc?

Reply:
You could do that. I tend to leave versions a bit more open so that the project using the gem can control this a bit more, in case the ROM API we are using doesn’t change across versions. But I could go either way.